server/app/lib/common.js

```diff
@@ -27,8 +27,8 @@ var getConfig = function (item, required, def) {
     }
   }
 
-  if (value.toString() === 'true') return true;
-  if (value.toString() === 'false') return false;
+  if (value !== undefined && value.toString() === 'true') return true;
```

Comment:
This one bit me too, in a different context.  Thanks for fixing it!

Reply:
Oh wait.  This doesn't catch the case when `value === null`.

Rather than do all these tricky comparisons, I'd rather say

    '' + value === 'true'

which is guaranteed not to throw an error.

Reply:
null.toString() doesn't throw an error though. But I can definitely integrate your changes.